sdk/translation/azure-ai-translation-document/tests/test_all_document_statuses.py

```diff
@@ -90,40 +56,163 @@ def test_list_statuses_with_pagination(self, client):
                 self._validate_doc_status(document, target_language)
 
 
-    @pytest.mark.skip("skip not exposed yet")
     @DocumentTranslationPreparer()
     @DocumentTranslationClientPreparer()
     def test_list_statuses_with_skip(self, client):
-        # prepare containers and test data
-        blob_text = b'blob text'
-        blob_data = [Document(data=blob_text), Document(data=blob_text), Document(data=blob_text),
-                     Document(data=blob_text), Document(data=blob_text), Document(data=blob_text)]
-        source_container_sas_url = self.create_source_container(data=blob_data)
-        target_container_sas_url = self.create_target_container()
-        docs_len = len(blob_data)
+        docs_count = 10
         skip = 2
         target_language = "es"
 
-        # prepare translation inputs
-        translation_inputs = [
-            DocumentTranslationInput(
-                source_url=source_container_sas_url,
-                targets=[
-                    TranslationTarget(
-                        target_url=target_container_sas_url,
-                        language_code=target_language
-                    )
-                ]
-            )
-        ]
-
         # submit and validate job
-        job_id = self._submit_and_validate_translation_job(client, translation_inputs, len(blob_data))
+        job_id = self._create_translation_job_with_dummy_docs(client, docs_count, language_code=target_language, wait=False)
 
         # check doc statuses
         doc_statuses = list(client.list_all_document_statuses(job_id=job_id, skip=skip))
-        self.assertEqual(len(doc_statuses), docs_len - skip)
+        self.assertEqual(len(doc_statuses), docs_count - skip)
 
         # iterate over docs
         for document in doc_statuses:
-            self._validate_doc_status(document, target_language)
\ No newline at end of file
+            self._validate_doc_status(document, target_language)
+
+
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    def test_list_submitted_jobs_filter_by_status(self, client):
+        docs_count = 10
+        target_language = "es"
+
+        # submit and validate job
+        job_id = self._create_translation_job_with_dummy_docs(client, docs_count, language_code=target_language, wait=False)
+
+        # list jobs
+        statuses = ["Running"]
+        doc_statuses = list(client.list_all_document_statuses(job_id, statuses=statuses))
+        self.assertIsNotNone(doc_statuses)
```

Comment:
Wondering if we might create a flaky test here since it's possible the job could finish with all documents before we get to this code block (failing on this assertion). just an idea - what if we create a document in the job that we know will get a "Failed" status and filter/validate on that?

Reply:
i know!
this is why i made sure to submit many docs for the job to make sure some are still 'Running'